tests/integration/docusaurus/tutorials/quickstart/quickstart_v1__1.py

```diff
@@ -0,0 +1,68 @@
+"""
+This workflow demonstrates how to use Great Expectations in a quick-start
+notebook environment. It is useful to explore and understand how Great Expectations
+works, using your own data.
+
+This workflow is not recommended for use in production environments because:
+1. it uses ephemeral assets
+2. it uses an iterative workflow of trying and refining expectations.
+"""
+
+# <snippet name="tutorials/quickstart/quickstart.py import_gx">
+import great_expectations as gx
+import great_expectations.expectations as gxe
+
+# </snippet>
+
+# Set up
+# <snippet name="tutorials/quickstart/quickstart.py get_context">
+context = gx.get_context()
+# </snippet>
+
+# Connect to data
+# <snippet name="tutorials/quickstart/quickstart.py connect_to_data">
+batch = context.sources.pandas_default.read_csv(
+    "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+)
+
+# What happened in the background?
+# Datsource: "default_pandas_datasource"
+# Asset: "#ephemeral_pandas_asset" -- CSVAsset (path lives here)
+# BatchConfig (Splitters): No Splitters
+# BatchOptions: (none needed)
+
+# TODO: ticket We can also use a SQL query as a data source
+context.sources.add_postgresql(
+    name="postgresql", connection_string="postgresql://localhost"
+)
+batch = context.sources.postgresql.query_batch("SELECT * FROM taxi LIMIT 1000")
+# </snippet>
```

Comment:
My original thought was to have the connect to data code, along with any related testing/verification, wrapped in a function that returns whatever is needed for the rest of the script (one function for each datasource demonstrated), and then call each of those functions once before proceeding with the rest of the example script.  

Because snippets automatically truncate excess spacing when they are imported into the docs, it won't matter that there is "extra" indentation from the snippet existing in a function rather than in the top level namespace, at least insofar as the docs are concerned.

Then, we would just have each snippet named something like: 
`snippet name="tutorials/quickstart/quickstart.py connect_to_data <specific datasource>"` and reference the appropriate `<specific datasource>` in each tab in the docs.